src/pretix/base/settings.py

```diff
@@ -359,6 +359,17 @@
 Best regards,
 Your {event} team"""))
     },
+    'mail_text_download_reminder': {
+        'type': LazyI18nString,
+        'default': LazyI18nString.from_gettext(ugettext_noop("""Hello,
+
+you bought a ticket for {event}.
+
+This is to remind you that your ticket is ready for downloading!
+
+Best regards,
+Your {event} team"""))
+    },
```

Comment:
We'd also need a setting ``mail_days_download_reminder`` as in "how many days before the event should we sent this". By default, it should be None, i.e. by default the mail should not be sent.